package.json

```
@@ -1,72 +1,28 @@
 {
-  "name": "haracho-approvers-dev",
-  "version": "0.0.0",
+  "name": "haracho.approvers.dev",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "ssh://git@github.com/approvers/haracho.approvers.dev.git",
```

```suggestion
  "repository": "https://github.com/approvers/haracho.approvers.dev.git",
```
どうせ指定するなら HTTPS のほうがよいかと. そもそも `"private": true` なので不要ですが